content/id/docs/reference/access-authn-authz/rbac.md

```diff
@@ -248,7 +249,7 @@ rules:
 Kamu juga dapat merujuk ke sumber daya dengan nama untuk permintaan tertentu melalui daftar `resourceNames`.
 Ketika nama dicantumkan, permintaan dapat dibatasi untuk setiap objek sumber daya.
 Berikut adalah contoh yang membatasi subjeknya hanya untuk melakukan `get` atau` update` pada sebuah
-ConfigMap bernama `my-configmap`:
+{{< glossary_tooltip term_id="ConfigMap" >}} bernama `my-configmap`:
```

Comment:
we don't have glossary ConfigMap in bahasa indonesia yet, feel free to localize this glossary when you have time